modules/graphics/mrblib/text.rb

```diff
@@ -0,0 +1,87 @@
+module Moon
+  class Text
+    extend TypedAttributes
+    include Shadable
+
+    attr_reader :w
+    attr_reader :h
+
+    attribute :shader,        Shader
+    attribute :string,        String
+    attribute :font,          Font
+    attribute :color,         Vector4
+    attribute :outline,       Integer
+    attribute :outline_color, Vector4
+    attribute :line_height,   Float
+
+    # @param [Font] font
+    # @param [String] string
+    def initialize(font, string = "")
+      set_font font
+      set_string string
+      @outline = 0
+      @line_height = 1.0
+      @w = 0
+      @h = 0
+      @shader = self.class.default_shader
+      @vbo = VertexBuffer.new(VertexBuffer::DYNAMIC_DRAW)
+      @color = Vector4.new(1, 1, 1, 1)
```

Comment:
I know we got rid of the color class, but would probably be nice to have a derivative of Vector3/4 that has initializers for RGB(A), HSL(A), and possibly hex codes: `Color.rgba(255, 255, 255, 1.0), Color.hex("#FFFFFF")`

That way we sorta match the CSS3 syntax.